src/components/ui/DataTable.js

```diff
@@ -0,0 +1,107 @@
+import { default as ReactDataTable } from 'react-data-table-component'
+import styled from 'styled-components/macro'
+
+import { RESOURCES } from '../entry/resources'
+
+const DataTableWrapper = styled.div`
+  .rdt {
+    &_TableHeadRow {
+      font-style: normal;
+      font-weight: bold;
+      font-size: 14px;
+    }
+    &_TableCell {
+      font-size: 14px;
+      &[data-column-id='scientific_name'] {
+        font-style: italic;
+      }
+    }
+  }
+`
+const TableLinkPreview = styled.img`
+  width: 20px;
+  height: 20px;
+  margin-right: 5px;
+`
+
+const customLinkSort = (rowOne, rowTwo) => {
+  if (rowOne.links.length > rowTwo.links.length) {
+    return 1
+  } else if (rowTwo.links.length > rowOne.links.length) {
+    return -1
+  }
+  return 0
+}
+
+const customDateSort = (rowOne, rowTwo) => {
+  const dateOne = new Date(rowOne.created_at)
+  const dateTwo = new Date(rowTwo.created_at)
+
+  if (dateOne < dateTwo) {
+    return -1
+  } else if (dateTwo > dateOne) {
+    return 1
+  } else {
+    return 0
+  }
+}
```

Comment:
You can actually just return `dateTwo - dateOne` rather than if/else. See compareFunction docs here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort